bundles/prometheus.0.14.0.substitutesfor/manifests/alertmanager.crd.yaml

```diff
@@ -0,0 +1,2398 @@
+apiVersion: apiextensions.k8s.io/v1beta1
```

Comment:
Also setting the `apiVersion` to `v1` for any CRDs would be good, since `v1beta1` will be deprecated in the next kube bump

Reply:
I can see the appeal in using production quality bundles for the tests but agree its not strictly necessary if we are just interested in testing the substitutes-for functionality

Reply:
Yeah I really want the most realistic test possible here in order to be as confident as possible wrt production impact.

@exdx
```
Also setting the apiVersion to v1 for any CRDs would be good, since v1beta1 will be deprecated in the next kube bump
```
should I change all the crd versions everywhere then?

Reply:
I would recommend it, yes. You will have to mess with the YAML CRD definitions a bit though for kube to accept them. You have a schema, but the v1 wants the schema to be on a per-version level versus at the top level. 

BUT at the same time, since these fixtures are never going to be applied on-cluster, idk if it's even worth the effort now that I'm thinking about it. Maybe not worth the effort unless others feel strongly.

Reply:
We may remove the ability to even add a v1beta1 CRD to a bundle though , so perhaps it does make sense actually

Reply:
> Yeah I really want the most realistic test possible here in order to be as confident as possible wrt production impact.

Ultimately, we're not changing anything related to CRD parsing though. Supporting enormous test fixtures is a burden on the team and, honestly,  it's hard to grok.